README.md

```diff
@@ -13,6 +13,18 @@ To automate the administration of `greenelab/scrum` issues, this repository reli
 Every day, Travis CI executes the commands in [`.travis.yml`](.travis.yml).
 As appropriate, **@scrum-lord** closes and opens issues to keep the scrum issues up to date.
 
+## Deployment Instructions
+
```

Comment:
Let's add a sentence here like: "Use these instructions to deploy a new instance of the scrumlord to manage scrum issues for a repository"